test/fulfillment.test.js

```diff
@@ -154,6 +166,18 @@ describe('Shopify#fulfillment', () => {
       .then((data) => expect(data).to.deep.equal(output.fulfillment));
   });
 
+  it('cancels a pending fulfillment of a FulfillmentOrder', () => {
```

Comment:
```suggestion
  it('cancels a fulfillment', () => {
```

Also please change the title of the previous test to `'cancels a fulfillment for a specific order ID'`